crates/cairo-profiler/src/trace_reader.rs

```diff
@@ -31,12 +31,12 @@ pub struct Sample {
 }
 
 impl Sample {
-    pub fn extract_measurements(
+    pub fn extract_sample_values(
         &self,
-        measurement_types: &[ValueType],
+        pprof_sample_units: &[pprof::ValueType],
```

Comment:
We shouldn't use pprof types for the control flow purposes imo, we should rely on SampleUnits for building the values